mapper.go

```diff
@@ -52,8 +52,16 @@ type metricMapping struct {
 	TimerType timerType         `yaml:"timer_type"`
 	Buckets   []float64         `yaml:"buckets"`
 	MatchType matchType         `yaml:"match_type"`
+	HelpText  string            `yaml:"help"`
 }
 
+type configLoadStates int
```

Comment:
What's this? Accidental rebase leftovers?

Reply:
yah I think that's it, lemme clean it up